national-water-model/xarray-water-model.py

```diff
@@ -0,0 +1,72 @@
+"""
+This example was adapted from https://github.com/dcherian/dask-demo/blob/main/nwm-aws.ipynb
+"""
+
+import coiled
+import dask
+import flox.xarray
+import fsspec
+import numpy as np
+import rioxarray
+import xarray as xr
+
+
+# optionally run with coiled run
+# coiled run --region us-east-1 --vm-type m6g.xlarge python nwm-aws.py
```

Comment:
Looks like the name of the script has changed

```suggestion
# coiled run --region us-east-1 --vm-type m6g.xlarge python national-water-model/xarray-water-model.py
```

Reply:
ah indeed, thanks!